Tribler/Core/TorrentChecker/torrent_checker.py

```diff
@@ -111,8 +59,8 @@ def initialize(self):
 
         self._reschedule_torrent_select()
 
-        self._checker_thread = TorrentCheckerThread(self)
-        self._checker_thread.start()
+        self.register_task(u"task_check_sockets", reactor.callLater(self._socket_selection_interval,
```

Comment:
@whirm @NielsZeilemaker The torrent selection interval is dynamic. This socket selection interval is fixed for the moment. I don't know if 0.3 is an appropriate number. Any idea?